internal/terraform/module.go

```diff
@@ -164,32 +177,43 @@ func getFileFormat(filename string) string {
 	}
 	return filename[last:]
 }
-func isFileFormatSupported(filename string) (bool, error) {
+func isFileFormatSupported(filename string, section string) (bool, error) {
```

Comment:
Since `section` is added as an argument, it's gonna be a bit useful to check if it's empty or not before doing anything further in the function.